build.sh

```diff
@@ -29,6 +29,39 @@ pushd "$ROOT_DIR/src/common/thirdparty/"
   bash build-redis.sh
 popd
 
+if [[ "${RAY_USE_NEW_GCS}" = "on" ]]; then
```

Comment:
should this be done in `build-redis.sh`? Or probably a new file `build-credis.sh`.